docs/superpowers/plans/2026-04-14-nebula-log-init-hardening.md

```
@@ -0,0 +1,852 @@
+/// Safe to call multiple times: once a dispatcher is already set, this
+/// returns a no-op [`LoggerGuard`] (see [`LogError::AlreadyInitialized`]
+/// for the `init_with` counterpart).
+///
+/// # Errors
+///
+/// Returns error if filter parsing fails or logger initialization fails.
+```
+
+In the top-of-crate docs (`//!` lines 83–90), append a sentence under the `OTEL_EXPORTER_OTLP_ENDPOINT` bullet:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)